<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectConfig</class>
 <widget class="QWidget" name="ProjectConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>479</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,1">
        <item>
         <widget class="QCheckBox" name="chkRequireDragenginePackage">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Require Drag[en]gine DragonScript package</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,0,0">
          <item>
           <widget class="QLineEdit" name="editPathInclude">
            <property name="toolTip">
             <string>A new include path to use for parsing files under given directory</string>
            </property>
            <property name="placeholderText">
             <string>A new include path to use for parsing files under given directory</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnPathIncludeSelect">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnPathIncludeAdd">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnPathIncludeRemove">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListView" name="listPathInclude">
          <property name="alternatingRowColors">
           <bool>false</bool>
          </property>
          <property name="uniformItemSizes">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
